### PR TITLE
Security

### PR DESCRIPTION
**Pull request by TomasJani**

### CODE REVIEW

**Comment by lukasmatta** [comment]
Hello, I'm trying this but endpoint /auth/login returns 405. Could you please look at it.

File: rest/src/main/java/cz/muni/fi/pa165/airportmanager/rest/security/AdminFilter.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebFilter(urlPatterns =  {"/flights"})
```
Comment (lukasmatta) [inline rest/src/main/java/cz/muni/fi/pa165/airportmanager/rest/security/AdminFilter.java:15]: Why allow only Admin to access /flights? We want any user to list all flights.

Comment (TomasJani) [reply rest/src/main/java/cz/muni/fi/pa165/airportmanager/rest/security/AdminFilter.java:15]: Sorry, this was just for testing.

**Comment by TomasJani** [comment]
> Hello, I'm trying this but endpoint /auth/login returns 405. Could you please look at it.

Please provide more information on what the problem is.
If I send post request at this url everithing is working.

`http://localhost:8080/pa165/rest/auth/login?name=admin&password=admin`

**Comment by lukasmatta** [comment]
You use session in this implementation, we can't use session as we won't use spring mvc but we'll have Angular frontend. I'm working on this and will update it.

**Pull request merged**